.github/workflows/checkcode.yml

```
@@ -78,7 +78,7 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: clippy
-        args: -- -D warnings
+        args: --tests -- -D warnings
```

```suggestion
        args: --workspace --tests --all-features -- -D warnings
```

To avoid further fixing :P